Client/Frontend/Browser/BrowserTrayAnimators.swift

```diff
@@ -103,7 +103,9 @@ private extension BrowserToTrayAnimator {
 
         // Force subview layout on the collection view so we can calculate the correct end frame for the animation
         tabTray.view.layoutSubviews()
-        tabTray.collectionView.scrollToItemAtIndexPath(NSIndexPath(forItem: bvc.tabManager.selectedIndex, inSection: 0), atScrollPosition: .CenteredVertically, animated: false)
+        if tabTray.collectionView.numberOfItemsInSection(0) > bvc.tabManager.selectedIndex {
+            tabTray.collectionView.scrollToItemAtIndexPath(NSIndexPath(forItem: bvc.tabManager.selectedIndex, inSection: 0), atScrollPosition: .CenteredVertically, animated: false)
+        }
```

Comment:
you'd have thought so, right? But apparently not and we crash here. I've only seen it on the iPad during testing, but there is a bug raised for it here:
https://bugzilla.mozilla.org/show_bug.cgi?id=1190871

It's possible that the changes I have made in https://github.com/mozilla/firefox-ios/pull/864 will resolve this too by preventing the data and the TayTray CollectionView getting out of sync in the first place.......